examples/nlp/language_modeling/conf/megatron_t5_config_finetune_glue_mnli.yaml

```diff
@@ -53,7 +53,7 @@ model:
       global_batch_size: 128
       micro_batch_size: 64
       shuffle: True
-      num_workers: 8
+      num_workers: 0
```

Comment:
Intensional ? It hurts training speed

Reply:
This is because the glue datasets are too small.